examples/png.dx

```diff
@@ -0,0 +1,116 @@
+'## Base 64 encoder
```

Comment:
It might be worth a header 
```suggestion

' # PNG
This shows how we can load and write a PNG image ....
first we need a Base64 encoder ...
## Base 64 encoder
```